index.html

```diff
@@ -2,17 +2,212 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Hello world</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://kit.fontawesome.com/1ada02b4f0.js" crossorigin="anonymous"></script>
+  <title>LEEHAR'S FASHION WEEK </title>
 </head>
 
 <body>
-    <header>
-        <h1 class="header">Hello world!</h1>
-    </header>
+  <header>
+    <div class="top-navbar">
+      <ul>
+        <li>
+          <a href="#"><i class="fa fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-facebook"></i></a>
+        </li>
+        <li><a href="#">Celebrities</a></li>
+        <li><a href="#">Vlog</a></li>
+        <li><a href="#"> Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-navbar flex-center">
+      <div class="logo">
+        <a href="#"><img src="./assets/creative-tailor.png" alt="fashion logo" /></a>
+      </div>
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./assets/hamburger.svg" alt="Menu Hamburger" />
```

Comment:
- [ ] The hamburger should be transparent or have the same color/pattern as the top section background. Currently, if i scroll down, I can still see a white background and this goes not give a good user experience
![image](https://user-images.githubusercontent.com/33067256/128323946-ac4c70f6-3f85-4e3d-a494-b37f9b39f112.png)

Reply:
The white background is used, so the hamburger menu doesn't overlap with the page text and cause readability issues.

Reply:
[Optional]
@rotimiazeez, I can see that you used a similar pattern with the background of the page. Which is ok for this project. (Since there is no design guideline for the menu) If you want to improve it further you can check [this article](https://coder-coder.com/background-image-opacity/).